.circleci/config.yml

```diff
@@ -131,7 +145,9 @@
       when: << pipeline.parameters.verify_stable_pacts >>
       jobs:
         - verify_pacts
-        - pact_broker_artifact
+        - notify_pact_consumers:
+            requires:
+              - verify_pacts
```

Comment:
Not sure how to get this to always run regardless of `verify_pacts` failing or passing.

Reply:
The only way I can think of is making a separate `verify_pacts_and_notify` job that does the notify command with `when: always`.